client/me/help/help-contact-form/index.jsx

```diff
@@ -116,7 +128,8 @@ module.exports = React.createClass( {
 				value: option.value,
 				title: option.label,
 				onClick: () => {
-					this.setState( { [ selectionName ]: option.value } )
+					this.setState( { [ selectionName ]: option.value } ),
+					this.trackClickStats( selectionName, option.value )
```

Comment:
Just one more minor thing, each line should be ended with a semicolon as this is a function body rather than an object:
this.setState( { [ selectionName ]: option.value } );
this.trackClickStats( selectionName, option.value );